docs/visual/faq.md

```diff
@@ -0,0 +1,214 @@
+<TabItem value="Drone">
+
+```js reference title="Screener Runner | Drone example"
+https://github.com/screener-io/screener-runner/blob/master/src/ci.js#L70-L76
+```
+
+</TabItem>
+<TabItem value="Bitbucket">
+
+```js reference title="Screener Runner | Bitbucket Pipelines example"
+https://github.com/screener-io/screener-runner/blob/master/src/ci.js#L62-L68
+```
+
+</TabItem>
+<TabItem value="Semaphore">
+
+```js reference title="Screener Runner | Semaphore example"
+https://github.com/screener-io/screener-runner/blob/master/src/ci.js#L78-L84
+```
+
+</TabItem>
+<TabItem value="Buildkite">
+
+```js reference title="Screener Runner | Buildkite example"
+https://github.com/screener-io/screener-runner/blob/master/src/ci.js#L94-L100
+```
+
+</TabItem>
+</Tabs>
+
+If these environment variables are NOT accessible in the build, the reason is most likely that the CI is running the test job inside a separate container, and the variables need to be passed into the container.
+
+### How do I add Screener Steps to Storybook Component Story Format (CSF) stories?
+Screener Storybook supports version 3.0 to 6.x (latest) and [CSF](https://storybook.js.org/docs/react/api/csf)-formatted stories. To see examples, refer to [GitHub Gist | Three ways to add Screener Steps to a React Component in Storybook to CSF](https://gist.github.com/screener-io/dfbd4b9aa5284e555ea83c936fc0b2ba).
+
+### How do I add Screener Steps to my Storybook project that uses React Hooks?
+React Hooks messes up using the Screener component because it executes its magic at runtime. So to use Screener Steps with React Hooks, you need to use Storybook Decorators to wrap the component story. Here is an example script: [Storybook + Screener Steps with Hooks](https://gist.github.com/screener-io/ada0e3c51137c907ba80728ad7fab23b).
+
+### Does Sauce Visual interact with knobs within stories in Storybook and test as needed?
+Screener-storybook does not automatically interact with knobs, as there would be potentially an infinite number of permutations. but if there's a UI state that the user wants to get to using knobs, then this is possible with Screener Steps.
+
+In the Storybook knobs add-on, there's a knobs UI where the user can copy a query string for the knobs configuration. They can copy this querystring, and use it with Screener Steps `executeScript` method to reproduce those knobs values.
+
+Click on the copy button in the lower right, then paste into an editor, then copy only the query-string portion (after the `?`) and paste that into a Screener step.
+
+```js title="Example"
+<Screener steps={new Steps()
+  .executeScript('window.location.href = "?path=/story/button&knob-Label=works"')
+  .snapshot('Knobs')
+  .end()
+}>
+```
+
+### How do I resolve `Error: Storybook object not found`?
+
+1. Run test with `--debug` flag enabled. For example: `npm run test-storybook -- --debug`.
+2. After running with debug flag, reach out to Sauce Labs Support Team and provide them with your log output.
+
```

Comment:
Feedback from Turan: add 'the' in "After running with THE debug flag..."... and then either "reach out to Sauce Labs' Support Team" or "reach out to the Sauce Labs Support Team".